Tasks/KubeloginInstallerV0/utils.ts

```diff
@@ -134,6 +146,36 @@ export async function unzipRelease(zipPath: string): Promise<string> {
   }
 }
 
+function getGithubEndPointToken(): string {
+  try{
+  const githubEndpoint = taskLib.getInput("gitHubConnection", false);
+  const githubEndpointObject = taskLib.getEndpointAuthorization(githubEndpoint, true);
+  let githubEndpointToken: string = null;
+
+  if (!!githubEndpointObject) {
+    taskLib.debug("Endpoint scheme: " + githubEndpointObject.scheme);
+
+    if (githubEndpointObject.scheme === 'PersonalAccessToken') {
+      githubEndpointToken = githubEndpointObject.parameters.accessToken
+    } else if (githubEndpointObject.scheme === 'OAuth'){
+      // scheme: 'OAuth'
+      githubEndpointToken = githubEndpointObject.parameters.AccessToken
+    } else if (githubEndpointObject.scheme === 'Token'){
+      // scheme: 'Token'
+      githubEndpointToken = githubEndpointObject.parameters.AccessToken
+    } else if (githubEndpointObject.scheme) {
+      throw new Error(taskLib.loc("InvalidEndpointAuthScheme", githubEndpointObject.scheme));
+    }
+  }
+
+  return githubEndpointToken;
+}
+catch(error)
+{
+  throw new Error(taskLib.loc("Failed to retrieve GitHub endpoint token:", error.message));
```

Comment:
Let's create less generic exception type here.

Reply:
handling the exception specifically